epictrack-web/src/components/workPlan/status/StatusView.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import NoDataEver from "../../shared/NoDataEver";
+import TrackDialog from "../../shared/TrackDialog";
+import StatusForm from "./StatusForm";
+import { WorkplanContext } from "../WorkPlanContext";
+
+const StatusView = () => {
+  const { status } = React.useContext(WorkplanContext);
+  const [showStatusForm, setShowStatusForm] = React.useState<boolean>(false);
```

Comment:
these perhaps will be needed in another component, like when clone status is pressed. Perhaps we can have these in the context? But no rush now, just your call or we can move forward on this now